### ISSUE TITLE
Errors while building bash_completion script on Mac

### ISSUE THREAD

**Issue by Loquats** [bazelbuild/bazel#6207]
### Description of the problem / feature request:
I cannot run `bazel build //scripts:bash_completion` to install shell completion. I'm looking at instructions here:
https://blog.bazel.build/2015/04/10/bash-completion.html

### Feature requests: what underlying problem are you trying to solve with this feature?
I'd like to use bash completion on Mac.

### Bugs: what's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.
```
git clone git clone git@github.com:bazelbuild/bazel.git
cd bazel
bazel build //scripts:bash_completion
```
The same errors show up on release 0.17.2:
```
git clone git clone git@github.com:bazelbuild/bazel.git
cd bazel
git checkout 0.17.2
bazel build //scripts:bash_completion
```

### What operating system are you running Bazel on?
MacOS High Sierra, Version 10.13.6

### What's the output of `bazel info release`?
release 0.7.0

### If `bazel info release` returns "development version" or "(@non-git)", tell us how you built Bazel.
n/a

### What's the output of `git remote get-url origin ; git rev-parse master ; git rev-parse HEAD` ?
git@github.com:bazelbuild/bazel.git
c361da0cb8b02e9616a13ffced10ef091a1f6219
c361da0cb8b02e9616a13ffced10ef091a1f6219

###  Have you found anything relevant by searching the web?
Sort of. Just this: https://github.com/bazelbuild/bazel/issues/1539

### Any other information, logs, or outputs that you want to share?
Here's the error output:
```
$ bazel build //scripts:bash_completion
ERROR: /Users/somehwere/bazel/tools/osx/xcode_version_flag.bzl:100:47: name 'configuration_field' is not defined.
ERROR: /Users/somewhere/bazel/src/BUILD:248:2: error loading package 'tools/osx': Extension 'tools/osx/xcode_version_flag.bzl' has errors and referenced by '//src:package-zip'.
ERROR: Analysis of target '//scripts:bash_completion' failed; build aborted: error loading package 'tools/osx': Extension 'tools/osx/xcode_version_flag.bzl' has errors.
INFO: Elapsed time: 0.376s
```

**Comment by rupertks** [comment]
Bazel 0.7.0 is quite old now (from October 2017). I tried to reproduce this with a newer version of Bazel, and it seems to work. Could you upgrade your Bazel version?

I used:

$ bazel info release
release 0.17.2-homebrew

**Comment by tetromino** [comment]
Bazel 0.7.0 is obsolete. You really ought to update.

We do not support using the old Bazel 0.7.0 binary for building anything from the current (0.17.x or 0.18.x) Bazel source tree. The error message you are encountering here is fixed by updating to a more recent version (0.17.2 definitely works).
